go.mod

```diff
@@ -255,4 +255,4 @@ require (
 	lukechampine.com/blake3 v1.1.7 // indirect
 )
 
-go 1.18
+go 1.19
```

Comment:
Do not change this, we must support the last 2 go versions.

Reply:
`GOMEMLIMIT` only exists in 1.19, 1.20 will release in ~<2 months, I think it's fine supporting 1.19 only (we don't even test 1.18 on CI).

The alternative is to use build tags to only set `GOMEMLIMIT` if you build with 1.19 or above.

Reply:
I think we agreed to maintain the code compatible with the two latest go versions, let's check with @ipfs/kubo-maintainers about this.

Reply:
> I think we agreed to maintain the code compatible with the two latest go versions

I know we do for libp2p. If we do for kubo we should start at least buiding with the previous version in CI.